src/components/SearchBlock.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Switcher } from "./Switcher";
+
+export class SearchBlock extends React.PureComponent {
```

Comment:
better to export one class from one module. just put SearchBlock and SearchInput classes in different files